CHANGELOG.md

```diff
@@ -3,6 +3,7 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## Unreleased
+- Add new applyToIDEA location for IDEA 2020.1.x and above on MacOs
```

Comment:
Please move it under `### Added` section. Sorry that it was not available initially.